boards/st/stm32f769i_disco/stm32f769i_disco.dts

```diff
@@ -74,6 +74,13 @@
 		sw0 = &user_button;
 		spi-flash0 = &mx25l51245g;
 	};
+
+	quadspi_memory_avail: memory-avail@90000000 {
```

Comment:
It would be nice to update board documentation to provide some details on how this could be used.